schemaregistry/schemaregistry.md

```diff
@@ -246,18 +333,28 @@ within the scope of the registry.
   - 1
   - 2
 
-#### id
+#### `id` (schema version id)
```

Comment:
should this be schema version uri? I think it's being mentioned like that in other parts of the document